lib/lightning/projects.ex

```diff
@@ -46,6 +46,38 @@ defmodule Lightning.Projects do
     {:ok, %{projects_deleted: projects_to_delete}}
   end
 
+  def perform(%Oban.Job{args: %{"type" => "data_retention"}}) do
```

Comment:
Hey @midigofrank I have made the `wipe_dataclips_for/1` function to take a project as argument so that you can integrate it more easily. I haven't touch the tests because they still work and they are more integration tests than unit tests. Please change them if you wanna make them unit test the `wipe_dataclips_for/1` function instead of the `perform/1` function.